pkg/generate/app/imagestreamlookup.go

```diff
@@ -8,14 +8,14 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
-	"github.com/openshift/origin/pkg/client"
 	imageapi "github.com/openshift/origin/pkg/image/apis/image"
+	imageclient "github.com/openshift/origin/pkg/image/generated/internalclientset/typed/image/internalversion"
 )
 
 // ImageStreamSearcher searches the openshift server image streams for images matching a particular name
 type ImageStreamSearcher struct {
-	Client            client.ImageStreamsNamespacer
-	ImageStreamImages client.ImageStreamImagesNamespacer
+	Client            imageclient.ImageStreamsGetter
```

Comment:
We should probably rename them to reflect what this client is.

Reply:
> We should probably rename them to reflect what this client is.

follow.  We're stacked up and these pulls conflict like crazy.

Reply:
Leave a TODO here to fix this.